### ISSUE TITLE
Easier way to rearrange list content

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#2510]
# Story
Followup to #2511 

As a site owner, I can order content in relation to each other (e.g. see weights for other content, drag and drop, or other option) so that prioritizing content is faster and has less room for human error.

# Proposed solution

Create a page, table, etc. specifically for ordering list content, or a view or context indicator on the form where referencing other content weights is available.

Main issues are that when setting a weight, there is no visible reference to how high or low it should be set, and that not all content needs to be weighted against each other, depending on the view.

**Comment by joewhitsitt** [comment]
Was this requested by Scott? If not, the admissions label shouldn't be added and this should be scoped with the the 80% solution in mind https://github.com/uiowa/uiowa/issues/2511

**Comment by cory-skeers** [comment]
Good catch, @joewhitsitt. Not a specific request--got the issue updated.
